### DISCUSSION TITLE
Information architecture for website discussion

### DISCUSSION THREAD

**Discussion by Asmita-si** [Swarnimashukla/Automatic-attendance-management-system#111] [General]
Hello, I have made the flowchart which I was assigned with. Please check it once:- https://whimsical.com/create-a-information-architecture-for-website-28-VS6jibM7UCYihDkjwL7DKE@2Ux7TurymN7t1JC8sWFZ . Let me know if I should make any changes.

**Comment by Swarnimashukla** [comment]
@Asmita-si  this looks good to me .. go for pr

**Comment by Asmita-si** [comment]
Okay. @Swarnimashukla can you please let me know how should I make the pr. Should I make it in image format, because it's just a link?

**Reply by Swarnimashukla** [reply]
Yes take screenshot and make a pr as image, keeping file name as  "IA_yourname"

**Reply by Asmita-si** [reply]
Okay. Thank you.

**Comment by Asmita-si** [comment]
@Swarnimashukla  some error has occurred in my PR. Can you please help me?
